db/migrate/20220404144241_add_gas_network_costs/data.csv

```diff
@@ -0,0 +1,503 @@
+GM0762,2150068.615
+GM0765,1678471.099
+GM0766,4054915.152
+GM0777,4424367.878
+GM0779,1350160.068
+GM0784,1528929.529
+GM0785,1168048.944
+GM0796,8226881.261
+GM0797,3026280.734
+GM0798,874116.5258
+GM0809,1186182.432
+GM0824,1517949.178
+GM0826,4104824.951
+GM0840,1358149.091
+GM0851,5629413.597
+GM0852,819334.6902
+GM0855,12871753.67
+GM0861,2652758.304
+GM0867,3444464.106
+GM0880,1215276.526
+GM0995,4151948.251
+GM1509,2225670.041
+GM1525,1758543.085
+GM1581,2804385.999
+GM1586,1926558.05
+GM1598,1293272.14
+GM1621,2159291.587
+GM1655,4852811.778
+GM1674,4903661.916
+GM1676,2294320.029
+GM1680,1698886.826
+GM1681,1786008.287
+GM1690,1507682.073
+GM1695,516869.7587
+GM1696,1377953.901
+GM1699,2033632.31
+GM1700,1865916.151
+GM1701,1426076.535
+GM1705,3403735.448
+GM1708,2348121.524
+GM1709,24463390.58
+GM1714,1795859.67
+GM1719,1405856.746
+GM1723,704492.3114
+GM1724,1175467.609
+GM1728,1317857.385
+GM1730,2088289.318
+GM1731,4272994.813
+GM1734,2751360.353
+GM1735,2650474.142
+GM1740,1814424.033
+GM1742,2123408.773
+GM1773,1022642.616
+GM1774,1682400.856
+GM1783,4554600.798
+GM1842,755679.3495
+GM1859,3935393.383
+GM1876,4443605.2
+GM1884,1148407.81
+GM1891,987121.0881
+GM1892,1798209.842
+GM1895,4614464.019
+GM1896,2015339.916
+GM1901,1733684.356
+GM1904,2913772.725
+GM1911,4955845.979
+GM1916,3078428.566
+GM1924,2404531.624
+GM1926,1616469.666
+GM1930,3931270.274
+GM1931,2501111.817
+GM1940,4286717.965
+GM1942,3516278.158
+GM1945,1943006.246
+GM1948,8975353.669
+GM1950,5674147.976
+GM1952,9242306.109
+GM1955,2436433.541
+GM1960,4733774.726
+GM1961,5571774.657
+GM1963,4006720.48
+GM1966,11551228.72
+GM1970,2742074.793
+GM1978,2222519.206
+HR,130482338.9
+HU,488673700.8
+IE,151887197.2
+IT,2607577369
+LT,121624052.2
+LU,46167219.51
+LV,25411737.65
+MT,77210.27909
+PL,970052293.6
+PT,144221556.6
+PV20,98766876.74
+PV21,47847563.22
+PV22,32419877.22
+PV23,73981998.46
+PV24,17732336.72
+PV25,141309054.6
+PV26,62551053.53
+PV27,183734631.7
+PV28,279456690
+PV29,262285179.8
+PV30,187147998.3
+PV31,195408592.9
+RES01,20095386.67
+RES02,15758624.34
+RES03,47457370.54
+RES04,16322316.44
+RES05,18907708.81
+RES06,47847563.22
+RES07,2404531.624
+RES08,115293794.3
+RES09,29671593.63
+RES10,4006720.48
+RES11,25485309.86
+RES12,51123145.4
+RES13,211731447.8
+RES14,11630764.9
+RES16,38426199.29
+RES17,145035599
+RES18,12279177.09
+RES19,61240227.86
+RES20,43272599.34
+RES21,6184022.398
+RES22,15224122.05
+RES23,44034747.04
+RES24,44629269.72
+RES25,18329567.1
+RES26,29999572.78
+RES27,43614537.97
+RES28,85537913.94
+RES29,30368421.37
+RES30,262285179.8
+RES31,134168523.7
+RGAMS01,2426248.902
+RGBRE01,49635.07601
+RGGD01,131233307.8
+RGGLD01,145812016.8
+RGHLD01,4001742.565
+RGNB01,116680151.8
+RGNH01,46020903.1
+RGNH02,19890019.59
+RGNH03,14519511.66
+RGNH04,8968108.387
+RGNH05,12111895.02
+RGNH06,15562024.15
+RGNH07,10754552.72
+RGNH08,5372428.27
+RO,499886106.4
+SE,67814929.53
+SI,45270756.66
+SK,237130280.8
+UK,38524413.73
+UKNI01,3395366816
```

Comment:
These numbers look strange to me.

| Code | Value |
| :-- | --: |
| UK | 38_524_413 |
| UKNI01 | 3_395_366_816 |

Here are a couple of other countries for comparison:

| Code | Value |
| :-- | --: |
| DE | 4_489_867_063 |
| FR | 2_317_109_824 |

@marliekeverweij Could you double-check the values, particularly for the countries, and ensure they make sense to you? For example, Finland and Slovakia have roughly comparable amounts of natural gas demand in the model but have wildly different costs. I'm afraid I don't know enough about the datasets to know if the values are correct.

The migration itself otherwise looks fine. :+1:

Reply:
Yes you are right: the values for UKNI01 and UK were mixed up!

I have double-checked all the values and cannot find any other strange value. For Finland and Slovakia the values are:
Finland - 3.8e10 MJ gas - 68755304.35 euros - 5% of the NL gas network costs
Slovakia - 1.31437e11 MJ gas - 237130280.8 euros - 16% of the NL gas network costs
@antw where did you see the wildly different costs?



</body>

</html>

Reply:
Thanks for checking, @marliekeverweij. I've merged this and will re-export the country datasets to ETSource shortly.